src/components/CreateNewBook.js

```diff
@@ -0,0 +1,9 @@
+const CreateNewBook = () => (
+  <form>
+    <input type="text" className="title" placeholder="title" />
+    <input type="text" className="author" placeholder="author" />
+    <button type="submit">submit</button>
+  </form>
+);
+
+export default CreateNewBook;
```

Comment:
**Nicely Done**
- [ ] Good effort working on the requested change by the previous Code Reviewer.